tools/run-tests.py

```diff
@@ -113,6 +113,24 @@ def run_core_tests(engine):
         raise Exception("wasm-test-core failed")
 
 
+@runner('wasm-new-test-core', default=False)
+def run_core_tests(engine):
+    TEST_DIR = join(PROJECT_SOURCE_DIR, 'test', 'wasm-spec', 'new-core')
```

Comment:
The new spec testsuite is added under `test/wasm-spec/new-core` directory.
`wasm-new-test-core` runner is newly added because JITC cannot support the whole new test suite now.
After JITC update, I'll merge this new testsuite into the original directory (`test/wasm-spec/core`)

Reply:
Would it be a good idea to add skiplist?

Reply:
Good idea! Currently, only 8 tests fail in JIT mode.
I'll update it